app/src/CoursePages/CoursePage.tsx

```diff
@@ -30,33 +30,96 @@ export const CoursePage: React.FC = () => {
         fetchData();
     }, [classNum]);
 
+    return (
+        <div className="coursepage">
+            <div className="topbox">
+                <CourseInfo classNum={ classNum! }
+                courseName={ course.length > 0 ? course[0].name : '...' }
+                desc={ course.length > 0 ? course[0].description : 'Loading...' }/>
+                <div className="rightflex">
+                    <h3 className="overallratings-header">Overall Ratings</h3>
+                    <div className="ratings-flexbox">
+                        <OverallRatingBox label="Difficulty" rating={ course.length > 0 ? course[0].rating_one : '?/5' }></OverallRatingBox>
```

Comment:
I think since OverallRatingBox has no body, you can put it as <OverallRatingBox (insert stuff) />

Reply:
Fixed.